theme/toolbox.less

```diff
@@ -259,6 +259,50 @@ div.blocklyTreeIcon span {
     background-color: #e4e4e4;
 }
 
+/*******************************
+    Inverted Toolbox
+*******************************/
+#root.hc .invertedToolbox div div {
```

Comment:
probably don't need the `div div` here, and if we do we should figure out why (selectivity fight?) -- default behavior in css selectors is that it will go down as far as needed no matter how many divs are between (if you want to explicitly say "only divs that are directly descendants" it requires an > operator like `.parent > .direct-child`)